FhirToDataLake/azurepipelines/nativeci.yml

```diff
@@ -0,0 +1,34 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+variables:
```

Comment:
how about creating a sub folder for native yaml files?

Reply:
Thanks. updated